projects/tsq-gallery/src/lib/components/tsq-gallery-viewer/tsq-gallery-viewer.component.ts

```diff
@@ -389,6 +450,9 @@ export class TSqGalleryViewerComponent {
       this.resetPosition();
       this.selectedFileIndex--;
       this.pdfLoading = false;
+
+      this.navigationDelay = true;
+      setTimeout(() => this.navigationDelay = false, 250);
```

Comment:
it's a placeholder, I havent find a solution to differentiate Swipe Event from WheelEvent, so I added a delay between transitions so that on a swipe it doesn't move more than once per swipe. 
I will see if perhaps I use hammerjs or if you have any other idea